start

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
```

Comment:
I'll move this to `bin/start`. BTW, since this gets cleaned up as soon as you leave the interactive test container, there's no `stop` script... hopefully that's not to confusing or inconsistent.  I debated calling this `start_dev_env` (same as what you have for the buildpack), but thought `start` is our standard name for setting up dev env?

Reply:
@BradleyBoutcher I've moved `start` to `bin/start`.
I also had to make a minor change to the `bin/test_unit` to get that to pass.
CI is green now.

Reply:
I've ran it through my head a few times from the perspective of "I'm a brand new contributor to this project", and I think this is a really solid change. Although it isn't immediately clear what "start" is referring to, I think keeping it in `bin` should avoid accidental usage, and the documentation in `CONTRIBUTING.md` should be clear enough for anyone new to the project to understand what it does. And, while we don't have a `stop` script, I don't think that would be an expectation for any contributors.

Reply:
On the topic of the "standard name for setting up a dev environment", I'm really not sure what the standard is. I picked `start_dev_env` just because I thought it encapsulates the entire functionality of the script, which is typically how I try name scripts. However, we use `start` differently in `secretless-broker` and elsewhere, so I'm not sure what the best name is. If we named our script directory `dev` instead of `bin`, I think `start` would be fine, but `bin/start` can be a little unclear depending on the clarity of documentation and the expectations of the user. But, I don't think you need to change the name of yours at all.

Reply:
I also prefer `start_dev_env` since it's clearer, but I'll leave it as `start` for now to be consistent with other repos. It's easy enough to switch if/when we have complete consensus.